src/bin/commands/clean.rs

```diff
@@ -93,4 +90,20 @@ impl Clean {
 
         Ok(())
     }
+
+    pub fn engine(&self) -> Option<&str> {
```

Comment:
I can get rid of most of these getters and just replace them with the raw fields now that we use a macro to extract the fields.